buildscripts/autogen_sources.py

```diff
@@ -116,9 +116,10 @@
     imports_start_line, import_end_line = min(imports_line_numbers), max(imports_line_numbers)
     import_section_text = ''.join(module_text_lines[imports_start_line: import_end_line + 1])
 
-    # read function templates for arithmetic and comparison operators from tempaltes module
-    func_template_arithmetic_binop = inspect.getsource(templates_module.sdc_pandas_series_operator_binop)
-    func_template_comparison_binop = inspect.getsource(templates_module.sdc_pandas_series_operator_comp_binop)
+    # read function templates for arithmetic and comparison operators from templates module
+    template_series_arithmetic_binop = inspect.getsource(templates_module.sdc_pandas_series_operator_binop)
+    template_series_comparison_binop = inspect.getsource(templates_module.sdc_pandas_series_operator_comp_binop)
+    template_str_arr_comparison_binop = inspect.getsource(templates_module.sdc_str_arr_operator_comp_binop)
```

Comment:
Shouldn't we also add some str arithmetic operations?

Reply:
Done, added operator.add and operator.mul for string series (as a separate overload though, not to complicate the common template)